frontend/src/employee-frontend/components/child-information/GuardiansAndParents.tsx

```diff
@@ -96,15 +56,42 @@ const GuardiansAndParents = React.memo(function Guardians({
               <Th>{i18n.personProfile.streetAddress}</Th>
             </Tr>
           </Thead>
-          <Tbody>{renderGuardians()}</Tbody>
+          <Tbody>
+            <UnwrapResult result={guardians}>
```

Comment:
Why not use `renderResult` here? Also, it would a good idea to wrap the whole `<table>` element in it since `<div>`s shouldn't be `<tbody>`'s children.

Reply:
Yeah, `renderResult` requires wrapping the whole table, that's why I initially used `UnwrapResult`. Now changed all three to use `renderResult`.